vendor/vendor.json

```diff
@@ -191,187 +351,196 @@
-        {
-            "checksumSHA1": "wYSNb+W2L5gJlGO8n6mGOGft8R8=",
-            "path": "github.com/ericchiang/k8s/apis/policy/v1alpha1",
-            "revision": "929412ddf58668dc5aa49e47fa822507f048d34b",
-            "revisionTime": "2017-04-20T15:30:38Z"
-        },
-        {
-            "checksumSHA1": "ioJ28pdUN6fDkOp8dT+Tg3HSqmk=",
-            "path": "github.com/ericchiang/k8s/apis/policy/v1beta1",
-            "revision": "929412ddf58668dc5aa49e47fa822507f048d34b",
-            "revisionTime": "2017-04-20T15:30:38Z"
-        },
-        {
-            "checksumSHA1": "UErnBsjjtmg3oYjLYU1S80oi3sk=",
-            "path": "github.com/ericchiang/k8s/apis/rbac/v1alpha1",
-            "revision": "929412ddf58668dc5aa49e47fa822507f048d34b",
-            "revisionTime": "2017-04-20T15:30:38Z"
-        },
-        {
-            "checksumSHA1": "Xl+Tm8ZOz0cMOrfLaQvu/lsWObU=",
-            "path": "github.com/ericchiang/k8s/apis/rbac/v1beta1",
-            "revision": "929412ddf58668dc5aa49e47fa822507f048d34b",
-            "revisionTime": "2017-04-20T15:30:38Z"
-        },
-        {
-            "checksumSHA1": "YyZyaF0k2NAQAZvsCOVdhAkfVU0=",
-            "path": "github.com/ericchiang/k8s/apis/settings/v1alpha1",
-            "revision": "929412ddf58668dc5aa49e47fa822507f048d34b",
-            "revisionTime": "2017-04-20T15:30:38Z"
-        },
-        {
-            "checksumSHA1": "vUc3mf0rE7CQ3B52wfrMDyspLgA=",
-            "path": "github.com/ericchiang/k8s/apis/storage/v1",
-            "revision": "929412ddf58668dc5aa49e47fa822507f048d34b",
-            "revisionTime": "2017-04-20T15:30:38Z"
-        },
-        {
-            "checksumSHA1": "7/oj1z0vG1pvRza+UuKQ6txdleI=",
-            "path": "github.com/ericchiang/k8s/apis/storage/v1beta1",
-            "revision": "929412ddf58668dc5aa49e47fa822507f048d34b",
-            "revisionTime": "2017-04-20T15:30:38Z"
-        },
-        {
-            "checksumSHA1": "mm5iTFmLQ6h98DKgiUuTCpHP9H4=",
-            "path": "github.com/ericchiang/k8s/runtime",
-            "revision": "929412ddf58668dc5aa49e47fa822507f048d34b",
-            "revisionTime": "2017-04-20T15:30:38Z"
-        },
-        {
-            "checksumSHA1": "Kk1UDqUx2Pr1LyvIIgcJBApTlCk=",
-            "path": "github.com/ericchiang/k8s/runtime/schema",
-            "revision": "929412ddf58668dc5aa49e47fa822507f048d34b",
-            "revisionTime": "2017-04-20T15:30:38Z"
-        },
-        {
-            "checksumSHA1": "LoxBND74egHIasOX6z98FeeW0zI=",
-            "path": "github.com/ericchiang/k8s/util/intstr",
-            "revision": "929412ddf58668dc5aa49e47fa822507f048d34b",
-            "revisionTime": "2017-04-20T15:30:38Z"
-        },
-        {
-            "checksumSHA1": "fobEKiMk5D7IGvCSwh4HdG1o98c=",
-            "path": "github.com/ericchiang/k8s/watch/versioned",
-            "revision": "929412ddf58668dc5aa49e47fa822507f048d34b",
-            "revisionTime": "2017-04-20T15:30:38Z"
-        },
+		{
```

Comment:
Interesting that indentation was off in the file. Did you edit it manually before?